test_framework/terraform/aws/sles/user-data-scripts/provision_rke2_server.sh.tpl

```diff
@@ -62,7 +62,9 @@ EOF
     systemctl restart systemd-sysctl
     useradd -r -c "etcd user" -s /sbin/nologin -M etcd -U
     cat << EOF >> /etc/rancher/rke2/config.yaml
-profile: "cis-1.23"
+profile: "cis"
```

Comment:
Can we also enable CIS hardening in K3S?

Reply:
> Can we also enable CIS hardening in K3S?

Theoretically, yes. However, my previous survey showed that `K3s` CIS hardening takes much more time to prepare and test. So, I plan to finish the `RKE2` part first. I will look into how to implement it for K3s after the `v1.11.0` release. 

cc @derekbit